spec/controllers/concerns/remotely_translatable_spec.rb

```diff
@@ -48,11 +49,29 @@
       end
     end
 
+    it "When api key has not been set in secrets should not detect remote_translations" do
+      allow(Rails.application.secrets).to receive(:microsoft_api_key).and_return(nil)
+      proposal = create(:proposal)
+      comment = create(:comment, commentable: proposal)
+
+      I18n.with_locale(:es) do
+        expect(detect_remote_translations([proposal, comment])).to eq []
+      end
+    end
+
     it "When defined in current locale should not detect remote_translations" do
       proposal = create(:proposal)
       comment = create(:comment, commentable: proposal)
 
       expect(detect_remote_translations([proposal, comment])).to eq []
     end
+
+    it "When resource class has not translates should not detect remote_translations" do
```

Comment:
"has not translates" seems to be a typo :thinking:. How about "is not translatable"?